classes/Group/Group.mjs

```diff
@@ -0,0 +1,117 @@
+import dbDriver from "../../database/driver.mjs"
+const database = new dbDriver("mongo")
+
+export default class Group {
+    constructor(groupId) {
+        this._id = groupId
+    }
+
+    async #loadFromDB() {
+        return database.getById(this.groupId, "Group")
+    }
+
+    async getMembers() {
+        return this.members ?? await this.#loadFromDB()
```

Comment:
`this.members` was not initialized in the constructor, I think this may break other methods assigning attributes to members if `this.members` is `null`. it looks like it broke `getByRole` but I haven't tested other methods yet